PWGLF/Tasks/Strangeness/lambdapolsp.cxx

```diff
@@ -500,13 +451,14 @@ struct lambdapolsp {
   void fillHistograms(bool tag1, bool tag2, const ROOT::Math::PxPyPzMVector& particle,
                       const ROOT::Math::PxPyPzMVector& daughter,
                       double psiZDCC, double psiZDCA, double centrality,
-                      double candmass, double candpt, double candeta)
+                      double candmass, double candpt, double candeta, double angle)
   {
 
     ROOT::Math::Boost boost{particle.BoostToCM()};
     auto fourVecDauCM = boost(daughter);
     auto phiangle = TMath::ATan2(fourVecDauCM.Py(), fourVecDauCM.Px());
-
+    if (angle < -1) {
+    }
```

Comment:
@prottayCMT @skundu692 Please do not ignore the linters warnings. This is a useless piece of code reported by the linter.